Sparkle/ja.lproj/Sparkle.strings

```diff
@@ -35,7 +35,7 @@
 "An important update to %@ is ready to install" = "%@ の重要なアップデートがインストールできます。";
 
 /* the unit for bytes */
-"B" = "B";
+"B" = "バイト";
```

Comment:
No, you don't need. It's the standard localization on OS X that only "bytes" is translated in Japanese.
For instance, NSDataFormatter localizes in the same way.

fig. A screen shot of Finder's Get Info (see the third line). 
![screen shot 2016-01-04 at 14 40 23](https://cloud.githubusercontent.com/assets/1165044/12084054/3d152016-b2f1-11e5-91a0-e7ec5900e89e.png)

However: I've just realized this is not "bytes" but "B". I'm actually not sure how Apple localizes "B", as "bytes" is normally used instead of "B" in such cases.
But anyway, I feel the localized "バイト" (means bytes) is more natural for here and KB, MB, GB are never localized in Japanese on OS X.